conf/defaults.yaml

```diff
@@ -195,7 +195,7 @@ worker.log.level.reset.poll.secs: 30
 # control how many worker receiver threads we need per worker
 topology.worker.receiver.thread.count: 1
 
-# Executor metrics reporting interval.
+# Executor metrics reporting interval, keep default val same with topology.builtin.metrics.bucket.size.secs
```

Comment:
Nit: Could you add to the comment the reason why these must be kept in sync?

Reply:
Because:
1. the internal metrics interval is set up by conf: topology.builtin.metrics.bucket.size.secs, we should keep sync with it
2. Now use ZK to store our metrics and the ui only show built in metrics, the metrics consumer's collecting interval are also default to 60 seconds

Reply:
Thanks, but I meant add a note to the comment. I can't tell from the current comment why the two properties need to be in sync, so I'd have to look up this PR to see why.

Reply:
Ok, got your idea.